docs/.vuepress/components/SearchBarDoc.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <StackLayout>
+    <SearchBar 
+      :text="searchPhrase" 
+      :hint="hint" 
+      :textFieldBackgroundColor="background"
+      :textFieldHintColor="hintColor"
+      @submit="onSubmit" 
+      @textChange="onTextChange($event)"
+      @clear="onClear"
+      @input="searchPhrase = $event"
+      id="searchbar__docs"
+      />
+    <div class="flex m-t-10">
```

Comment:
Could you convert div to StackLayout or any Layout?

Reply:
done